models/commentsModel.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection.js");
+const selectCommentsByArticleId = (article_id) => {
+  const query_str = `SELECT comment_id,votes,created_at,author,body FROM comments WHERE article_id =$1 ORDER BY created_at DESC;`;
+  return db.query(query_str, [article_id]).then(({ rows: comments }) => {
+    console.log(comments);
+    if (comments.length === 0) {
```

Comment:
be careful here - this tells us that there are no comments but it doesn't tell us if that is because the article doesn't exist. We'll need to check this first.